test/test_autograd.py

```diff
@@ -10777,6 +10777,27 @@ def fn2(x1, x2):
         torch.autograd.gradcheck(fn2, [inp_r, inp_c], check_forward_ad=True)
         torch.autograd.gradcheck(fn2, [inp_c, inp_r], check_forward_ad=True)
 
+    def test_set_multithreading_enabled_as_context_manager_and_function(self):
+        # Test as a context manager
+        with torch.autograd.set_multithreading_enabled(False):
+            self.assertFalse(torch._C._is_multithreading_enabled())
```

Comment:
you can use the public version in the tests! `torch.autograd.is_multithreading_enabled()`

Reply:
Cool thank you, appreciate the quick review!